### DISCUSSION TITLE
How can I enable 'show-language' feature of prism?

### DISCUSSION THREAD

**Discussion by CCandle** [hexojs/hexo#5011] [Question / Answer] [answered]
for now I'm writing something about latex, so I switched to prism to highlight my code.
and I fond that primjs support to show language of my code, but how?
I tried
```yml
show_language: true
```
but obviousl it's not working.
since [the documet of hexo](https://hexo.io/docs/syntax-highlight.html#preprocess) says:
> + [Show Languages](https://prismjs.com/plugins/show-language/): Hexo will always have data-language attribute added as long as language is given for the code block.

how can I enable it?

**Comment by SukkaW** [comment]
Hexo's built-in prism will always have the `data-language` attribute present, and there is no option to disable `data-language` from Hexo's output.

You still have to include the prism plugin and its CSS in your theme in order for this feature to work.

**Reply by CCandle** [reply]
yes I noticed that there is an atrribute `data-language`, but for now there is no language shown on my website. how can I enable that? 
![image](https://user-images.githubusercontent.com/45314580/177489751-02097720-d44e-41d3-a465-489052d88402.png)
(I notice that the theme seems doesn't work for now, that doesn't matter)

**Reply by stevenjoezhang** [reply, marked as answer]
You need to load corresponding front-end plugins. Please contact the developer of the theme you're using for help
